pkg/controller/dataimportcron-controller.go

```diff
@@ -1508,13 +1560,21 @@ func (r *DataImportCronReconciler) newDataSource(cron *cdiv1.DataImportCron) *cd
 	return dataSource
 }
 
-// Create DataVolume name based on the DataSource name + prefix of the digest sha256
+// Create DataVolume name based on the DataSource name + prefix of the digest
 func createDvName(prefix, digest string) (string, error) {
-	fromIdx := len(digestPrefix)
-	toIdx := fromIdx + digestDvNameSuffixLength
-	if !strings.HasPrefix(digest, digestPrefix) {
```

Comment:
isnt it better to just do 3 `if` one per suffix since currently we only have 2 prefix options?
```
if strings.HasPrefix(digest, digestSha256Prefix) {
    digestPrefix = digestSha256Prefix
} else if strings.HasPrefix(digest, digestUIDPrefix) {
    digestPrefix = digestUIDPrefix
} else {
    return "", errors.Errorf("Digest has no supported prefix")
}
```

Reply:
Agree:)